android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt

```diff
@@ -192,117 +190,136 @@ fun HomeListChildView(
 				verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.s)),
 				horizontalAlignment = Alignment.CenterHorizontally,
 			) {
+
 				item {
-					LazyRow(
-						modifier = Modifier
-							.fillMaxWidth(1.0f),
-						verticalAlignment = Alignment.CenterVertically,
-						horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.xs)),
-					) {
-						item {
-							ButtonWithIcon(
-								modifier = Modifier
-									.width(dimensionResource(id = R.dimen.xl))
-									.height(dimensionResource(id = R.dimen.l)),
+					Column() {
+						LazyRow(
+							modifier = Modifier
+								.fillMaxWidth(1.0f),
+							verticalAlignment = Alignment.CenterVertically,
+							horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.xs)),
+						) {
+							item {
+								ButtonWithIcon(
+									modifier = Modifier
+										.width(dimensionResource(id = R.dimen.xl))
+										.height(dimensionResource(id = R.dimen.l)),
 //								TODO: make this dark mode conscious
-								iconId = R.drawable.tune_round_black_24,
-								onClick = {
-									filterType.value = FilterType.ALL
-									coroutineScope.launch {
-										isBottomSheetOpen = true
+									iconId = R.drawable.tune_round_black_24,
+									onClick = {
+										filterType.value = FilterType.ALL
+										coroutineScope.launch {
+											isBottomSheetOpen = true
 
-									}
-								},
-								contentDescription = stringResource(id = R.string.filter_menu),
-							)
-						}
-						item {
-							FilterButton(
-								filterName = stringResource(id = R.string.location),
-								onClick = {
-									filterType.value = FilterType.LOCATION
-									coroutineScope.launch {
-										isBottomSheetOpen = true
+										}
+									},
+									contentDescription = stringResource(id = R.string.filter_menu),
+								)
+							}
+							item {
```

Comment:
Why don't you do something like this here?
```
listOf(FilterType.LOCATION).map { 
        item {
	        FilterButton(
		        filterName = stringResource(id = it.nameStringId),
		        onClick = {
			        filterType.value = it
			        coroutineScope.launch {
				        isBottomSheetOpen = true
        
			        }
		        },
	        )
        }
}
```
You can change the `FilterType` to also contain the `stringRes` for `filterName`